rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -165,3 +172,103 @@ impl Run {
         Ok(())
     }
 }
+impl Layout {
+    pub fn new() -> LiveSplitLayout {
+        let mut layout = LiveSplitLayout::new();
+        layout.push(Component::Title(Title::new()));
+        layout.push(Component::Splits(Splits::new()));
+        layout.push(Component::DetailedTimer(Box::new(DetailedTimer::new())));
+        layout.push(Component::PreviousSegment(PreviousSegment::new()));
+        layout.push(Component::PossibleTimeSave(PossibleTimeSave::new()));
+        layout.push(Component::CurrentPace(CurrentPace::new()));
+        layout.push(Component::PbChance(PbChance::new()));
+        layout.push(Component::SumOfBest(SumOfBest::new()));
+        layout
+    }
+    fn settings() -> GeneralLayoutSettings {
+        LIVESPLIT_STATE.lock().unwrap().layout.general_settings().clone()
+    }
+    fn settings_mut() -> MappedMutexGuard<'static, GeneralSettings> {
+        MutexGuard::map(LIVESPLIT_STATE.lock().unwrap(), |x| x.layout.general_settings_mut())
+    }
+}
+#[rebo::function("Tas::timer_layout_get_best_segment_color")]
```

Comment:
Does it make sense to expose `LiveSplit::get_best_segment_color` or something like that for all current `timer_*` functions to scope them better and improve naming?